packages/ui-components/src/components/form.tsx

```diff
@@ -7,7 +7,8 @@ import { ITranslator, nullTranslator } from '@jupyterlab/translation';
 import { JSONExt, ReadonlyJSONObject } from '@lumino/coreutils';
 
 import Form, { FormProps, IChangeEvent } from '@rjsf/core';
-
+import { WidgetProps } from '@rjsf/utils';
```

Comment:
You need to add those imports:

```suggestion
import { 
  ariaDescribedByIds,
  schemaRequiresTrueValue,
  FormContextType,
  RJSFSchema,
  StrictRJSFSchema,
  WidgetProps
} from '@rjsf/utils';
```